spec/services/dashtag/environment_service_spec.rb

```diff
@@ -158,29 +158,35 @@ def header_title_helper_for(env_title_value)
         expect(EnvironmentService.hashtag_array).to be_empty
       end
 
-      it 'should parse EnvironmentService.hashtag into an array' do
+      it 'should parse EnvironmentService.hashtag into an array of arrays' do
         ENV["HASHTAGS"] = "yolo|dance|christmas"
-        expected_array = ['yolo', 'dance', 'christmas']
+        expected_array = [['yolo'], ['dance'], ['christmas']]
+        expect(EnvironmentService.hashtag_array).to eq(expected_array)
+      end
+
+      it 'should parse EnvironmentService.hashtag with ANDS into an array of arrays' do
+        ENV["HASHTAGS"] = "yolo|dance|christmas+candy|turtles"
```

Comment:
I know I said to use `+` to delimit compound hashtags, but I was thinking we should make this more intuitive. Inline with #100, we may want to use `and` or `&`, instead. So if we fixed #100 and used `and`, this line would look like

```ruby
ENV["HASHTAGS"] = "#yolo, #dance, #christmas & #candy, #turtles"
```